addons/versioning/XEH_postInitServer.sqf

```diff
@@ -18,23 +18,35 @@ FUNC(paranoid) = {
     };
 };
 
-QGVAR(versions_serv) addPublicVariableEventHandler { (_this select 1) call FUNC(paranoid) };
+QGVAR(versions_serv) addPublicVariableEventHandler {(_this select 1) call FUNC(paranoid)};
 
 // Missing Modfolder check
 FUNC(handleMismatch) = {
     params ["_machine","_mod"];
-    [format["%1 - Not running! (Machine: %2)", _mod, _machine], QUOTE(COMPONENT), [CBA_display_ingame_warnings, true, true]] call CBA_fnc_debug;
+    [format ["%1 - Not running! (Machine: %2)", _mod, _machine], QUOTE(COMPONENT), [CBA_display_ingame_warnings, true, true]] call CBA_fnc_debug;
 };
 
 QGVAR(mismatch) addPublicVariableEventHandler { (_this select 1) call FUNC(handleMismatch) };
 
 private "_str";
-_str = 'if(isServer)exitWith{};if (isNil "CBA_display_ingame_warnings") then { CBA_display_ingame_warnings = true; };LOG("cba_versioning_check");0 = objNull spawn { sleep 1; sleep 1; _func={GVAR(mismatch)=[format["%2 (%1)",name player, player],_this];publicVariable QGVAR(mismatch);_this spawn{_t=format["You are missing the following mod: %1",_this];diag_log text _t;sleep 2;if (CBA_display_ingame_warnings) then {player globalChat _t}}};';
+
+_str = 'if(isServer)exitWith{};if(isNil "CBA_display_ingame_warnings")then{CBA_display_ingame_warnings = true;};LOG("cba_versioning_check");0=objNull spawn{sleep 1;sleep 1;_func={GVAR(mismatch)=[format["%2 (%1)",name player,player],_this];publicVariable QGVAR(mismatch);_this spawn{_t=format["You are missing the following mod: %1",_this];diag_log text _t;sleep 2;if(CBA_display_ingame_warnings)then{player globalChat _t}}};';
+
 [GVAR(versions_serv), {
     _cfg = (configFile >> "CfgSettings" >> "CBA" >> "Versioning" >> _key);
-    _addon = if (isClass _cfg) then { if (isText (_cfg >> "main_addon")) then { getText (_cfg >> "main_addon") } else { _key + "_main" }; } else { _key + "_main" };
+    _addon = if (isClass _cfg) then {
+        if (isText (_cfg >> "main_addon")) then {
+            getText (_cfg >> "main_addon");
+        } else {
+            _key + "_main";
+        };
+    } else {
+        _key + "_main";
```

Comment:
Those three above are return values and to indicate that the semi-colon is usually left out.